index.html

```diff
@@ -54,8 +54,204 @@ <h4 class="social-media-intro">LET’S CONNECT</h4>
+            <li class="tag-btn"> <button type="button"  class="btn-small">javascript</button></li>
+        </ul>
+
+        <button type="button" class="btnNormal">See Project</button>
+            
+        </div>
+
+        <!-- card ends here -->
+
+        <!-- single card start here -->
+        
+        <div class="card fouth-card"> 
+            <img src="cardFourImage.svg" alt="" class="card-img">
+
+        <h2 class="project-title">Multi-Post
+            Stories</h2>
+
+        <ul class="workInfo">
+            <li class="workInfoDetails first-work"><a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a> <img class="counter" src="Counter.svg" alt=""> </li>
+            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">Back End Dev</a> <img class="counter" src="Counter.svg" alt=""> </li>
+            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">2015</a></li>
+        </ul>
+
+        <p class="details-sec">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="tag-btn-container">
+            <li class="tag-btn"> <button type="button"  class="btn-small">html</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">css</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">javascript</button></li>
+        </ul>
+
+        <button type="button"  class="btnNormal">See Project</button>
+            
+        </div>
+
+        <!-- card ends here -->
+
+
+        <!-- About session starts -->
+        <section class="about">
+            <h2 class="intro">
+                About
+                Myself
+            </h2>
+    
+            <p class="about-me-details">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+    
+    
+    
+            <h4 class="social-media-intro">LET’S CONNECT</h4>
+    
+            
+                <ul class="social-media-icon">
+                   <li class="social-media"><a class="list-style" href="#" ><img src="tweeter.svg"        alt="" ></a></li>
+                   <li class="social-media"><a class="list-style" href="#" ><img src="Linkedin-icon.svg"  alt="" ></a></li> 
+                   <li class="social-media"><a class="list-style " href="#" ><img src="mediaThree.svg"     alt="" ></a></li> 
+                   <li class="social-media"><a class="list-style" href="#" ><img src="mediaFour.svg"      alt="" ></a></li> 
+                   <li class="social-media"><a class="list-style" href="#" ><img src="mediaFive.svg"      alt=""></a></li>
+    
+                </ul>
+
+                <button type="submit"  class="btnNormal">Get my resume</button>
+
```

Comment:
- [ ] Please apply the hovering on the get my resume button 

![image](https://user-images.githubusercontent.com/74744787/164707680-13ad4cc7-f4f4-49d3-96c8-a365a80bf33c.png)
![image](https://user-images.githubusercontent.com/74744787/164707752-8d0e81ca-88e2-4fa0-a95a-1240253890a9.png)